curriculum/challenges/english/10-coding-interview-prep/rosetta-code/sorting-algorithms-bead-sort.md

```diff
@@ -8,11 +8,14 @@ dashedName: sorting-algorithmsbead-sort
 
 # --description--
 
-Sort an array of positive integers using the [Bead Sort Algorithm](https://en.wikipedia.org/wiki/Bead_sort).
+Sort an array of positive integers using the Bead Sort Algorithm.
 
-A *bead sort* is also known as a *gravity sort*.
+A *bead sort* is also known as a *gravity sort*. The bead sort operation can be compared to the manner in which beads slide on parallel poles, such as on an abacus. However, each pole may have a distinct number of beads.
```

Comment:
I don't think we can use wikipedia's text without attribution.
https://en.wikipedia.org/wiki/Wikipedia:Text_of_Creative_Commons_Attribution-ShareAlike_3.0_Unported_License